### ISSUE TITLE
Trend box disappears on Notifications page , does not reappear without reload

### ISSUE THREAD

**Issue by zetorplugger** [Bl4Cc4t/GoodTwitter2#98]
This was not an issue with 0.0.18 version but now when I click on Notifications  the trend box either moved to the right (with only 5 trends showing) or disappears altogether and then on Home does not appear either
other times if I click on Messages  the trend box only appears on the right side when going back to Home

![trend](https://user-images.githubusercontent.com/66390092/84063640-a299eb80-a9c1-11ea-9bfb-228ed5dc0d60.gif)

this function doesn't work on the Home, Notification pages all the time as it should
and it also doesn't work  if I search for something

    // handle trends (move and show10)
    function handleTrends() {
    let w = window.innerWidth
    let trends = div[data-testid=sidebarColumn] div:nth-child(4) > div[data-testid=trend], .gt2-left-sidebar div:nth-child(4) > div[data-testid=trend]

     waitForKeyElements(trends, function() {
      // move trends
      if (GM_getValue("opt_gt2").leftTrends
           && ((!GM_getValue("opt_gt2").smallSidebars && w > 1350)
             || (GM_getValue("opt_gt2").smallSidebars && w > 1230)))
      {
         if ($(".gt2-trends").length) $(".gt2-trends").remove()
        $(trends).parents("section").parent().parent().parent()
         .detach().addClass("gt2-trends")
         .appendTo(".gt2-left-sidebar")
       }

      // show 10 trends
       if (GM_getValue("opt_gt2").show10Trends)
       {
           if ($(trends).parent().parent().find("> div").length == 7)
           {
           $(trends).parent().parent().find("> div[role=button]").click()
           }
       }
     })

    }

**Comment by zetorplugger** [comment]
Does anyone else have this problem?

**Comment by Bl4Cc4t** [comment]
I think I fixed that in 340f29c556737a33d5dadb53974dfd532547e721, forgot to tag this issue.
Note, this is not related to #87/#29
